src/v2/mod.rs

```diff
@@ -2443,6 +2443,31 @@ impl Client {
         }
         last_found.ok_or(QueryError::NotFound)
     }
+
+    /// Get all bakers in the reward period of a block.
+    /// This endpoint is only supported for protocol version 6 and onwards.
+    /// If the protocol does not support the endpoint then an
+    /// 'IllegalArgument' error is returned.
```

Comment:
```suggestion
    /// [`IllegalArgument`](tonic::Code::InvalidArgument) is returned.
```

Let's not use haddock markup in Rust, it does not work.